examples/starter/variables.tf

```diff
@@ -26,10 +26,6 @@ variable "region" {
   type        = string
   description = "MongoDB Atlas Cluster Region, must be a region for the provider given"
 }
-variable "mongodbversion" {
```

Comment:
why remove this?

Reply:
because it was not important for the example, so I tried to keep the cluster creation only with relevant attributes for the examples